models/events/TestTakerRemovedEvent.php

```diff
@@ -22,11 +22,32 @@
 
 namespace oat\taoTestTaker\models\events;
 
+use oat\tao\model\webhooks\WebhookSerializableEventInterface;
+
 /**
  * Class TestTakerRemovedEvent
  * @package oat\taoTestTaker\models\events
  */
-class TestTakerRemovedEvent extends AbstractTestTakerEvent
+class TestTakerRemovedEvent extends AbstractTestTakerEvent implements WebhookSerializableEventInterface
 {
     const EVENT_NAME = __CLASS__;
+
+    /**
+     * @inheritDoc
+     */
+    public function getWebhookEventName()
+    {
+        return 'test-taker-removed';
```

Comment:
i would use a constant

Reply:
It is used only here, so I do not see the value, but I created.